### ISSUE TITLE
speakers shortcuts

### ISSUE THREAD

**Issue by fpereira1** [oTranscribe/oTranscribe#50]
At the moment we have `ctrl+J` to insert the current time.

It would be nice to have shortcuts for a list of speakers, like `ctrl+1` for speaker 1 and so on.

If other people find this useful I could submit a PR.

**Comment by akujugagila** [comment]
I agree and in the case of 2 speakers, there should be option to automatic alternate 2 spaeakers. I support the PR.
